### PR TITLE
LUN-65: Change appid for ios

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: ios/Lunes.xcworkspace/contents.xcworkspacedata [hunk 1]
```diff
@@ -2,9 +2,12 @@
 <Workspace
    version = "1.0">
    <FileRef
-      location = "group:LunesAppUI.xcodeproj">
+      location = "group:/Volumes/external/projects/lunes-app-steffi/lunes-app/ios/Lunes.xcodeproj">
```
Comment (klinzo) [inline ios/Lunes.xcworkspace/contents.xcworkspacedata:5]: This looks wrong

File: ios/Podfile [hunk 2]
```diff
@@ -23,17 +23,17 @@ target 'LunesAppUI' do
...
   # Note that if you have use_frameworks! enabled, Flipper will not work and
   # you should disable these next few lines.
-  use_flipper!
-  post_install do |installer|
-    flipper_post_install(installer)
-  end
+  # use_flipper!
+  # post_install do |installer|
+    #flipper_post_install(installer)
+  #end
```
Comment (klinzo) [inline ios/Podfile:29]: Why did you comment this?

**Review by klinzo — APPROVED** [review]

File: ios/Podfile (hunk 2 above)
Comment (SteffiStuffel) [reply ios/Podfile:29]: breaks the app. Will fix it later or is it really necessary to have it now?

Comment (klinzo) [reply ios/Podfile:29]: No, fix later is fine.

**Pull request merged**